Cargo.toml

```diff
@@ -10,14 +10,15 @@ keywords = ["lru", "concurrent", "cache"]
 categories = ["caching", "concurrency", "data-structures"]
 readme = "README.md"
 exclude = ["fuzz"]
+rust-version = "1.63"
 
 [features]
-default = ["dep:ahash"]
+default = ["ahash", "parking_lot"]
```

Comment:
Curious again, why not use the `dep:` prefix?

Reply:
[Features with the `dep:` prefix are implicitly created by Cargo](https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies) just for the purpose of being referenced by other features, and can't be toggled on by dependent crates after setting `default-features = false`. Moreover, these feature names for optional dependencies follow the conventions established by well-known crates, such as `tokio`.